src/components/NavBarLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+type Props = {
+  title: string,
+  redirectTo: string,
+};
+
+export const NavBarLink:React.FC<Props> = ({ title, redirectTo }) => (
+  <NavLink
+    className={({ isActive }) => cn('navbar-item',
+      { 'has-background-grey-lighter': isActive })}
+    to={`${redirectTo}`}
```

Comment:
The use of template literals is unnecessary when there is no computation or variable inside. The `redirectTo` prop can be passed directly without wrapping inside template literals.